src/components/ADempiere/Form/ProductInfo/productList.vue

```diff
@@ -74,7 +107,11 @@
         width="100"
       >
         <template slot-scope="scope">
-          {{ formatQuantity(scope.row.quantityOnHand) }}
+          {{
```

Comment:
Just can see change for soiurce code with the same functionality, please let it without changes